tmt/base.py

```diff
@@ -980,6 +989,37 @@ def _save_tree(self, tree):
         except tmt.utils.MetadataError:
             self._use_default_plan()
 
+    def _copy_tree_root(self):
+        """ Copy tree root to the workdir, under 'tree' directory """
+        tree_root = self.tree.root or '.'
+        workdir_tree = self.workdir_tree
+        # Use git worktree to initialize the tree
+        if os.path.exists(os.path.join(tree_root, '.git')):
+            try:
+                self._git_worktree(tree_root)
+                # Make sure tree is not is not created again in dest
+                workdir_tree = f'{workdir_tree}/'
+            except tmt.utils.RunError as error:
+                self.debug(f'Not using git worktree due to: {error}')
+                pass
+
+        # Sync additional data, except .git
+        self.run(
+            f'rsync -arv --exclude .git {tree_root}/ {workdir_tree}',
+            cwd=self.workdir)
+
+    def _git_worktree(self, tree_root):
+        """ Create git worktree under 'tree' directory """
+        self.debug(
+            f"Create git worktree from '{tree_root}' to '{self.workdir_tree}'")
+        # Branch must be set, it actually names the worktree
+        branch_name = self.workdir.replace('/', '-').lstrip('-')
+        # Make sure command runs against existing an existing directory
+        self.run(
+            f'git -C {tree_root} worktree '
```

Comment:
Can be simplified:
```suggestion
            f'git worktree '
```

Reply:
we decided to ditch `git worktree` in this release ... so this code was removed